server/controllers/activitypub/lib/format_items_activities.js

```diff
@@ -49,11 +49,13 @@ export const findFullRangeFromActivities = activitiesDocs => {
 const itemsWithinActivityRange = (since, until) => item => item.created > since && item.created < until
 
 const buildLinkContentFromItem = item => {
-  return {
+  const content = {
     text: item.snapshot['entity:title'],
     url: `${host}/items/${item._id}`,
-    details: `\n${item.details}`,
   }
+  const { details } = item
+  if (details) Object.assign(content, { details })
```

Comment:
[non blocking] a simpler syntax is possible
```suggestion
  if (details) content.details = details
```

Reply:
done in d7f5101c6